.eslintrc.js

```diff
@@ -25,7 +31,7 @@ module.exports = {
   },
   settings: {
     react: {
-      version: 'detect',
+      version: reactVersion,
```

Comment:
This setting was set to 'detect' which did not work because React is not installed in the root. Eslint then guesses that we’re using the latest version of React, which worked fine up until React 17. The only drawback from fetching the version from the core-react package, is that we now assume we’re using the same version of React in all the projects.